### ISSUE TITLE
Definitions for the final exam.

### ISSUE THREAD

**Issue by bharat1097** [cisc3410/4-review#5]
Anyone here write down all the definitions @rivlev mentioned at the end of the review lecture? If so, could you please post them here.

Thanks, 
Bharat

**Comment by brmosk** [comment]
Machine Learning, Overfitting, Regularization, Cross Validation, Gradient Descent, Non-parametric model, Support vector in SVM, kernel trick.
